src/components/CategoryHeader.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Router from "react-router";
+import mui from "material-ui";
+
+export default class CategoryHeader extends React.Component {
+
+    render(): React.Element {
+        return (
+            <div className="CategoryHeader">
+                <div className="found">
+                    I found 25 housing services for men
+                    ages 25-54 in Brunswick
+                </div>
+                <a className="search-link" href="#">
+                    Try a different search
+                </a>
+            </div>
+        );
+    }
```

Comment:
This probably doesn't need its own class IMO. Also you can get (some of) the live data for this field from `this.state.meta` on the category page.